### PR TITLE
[FLOC-2131] Boto configuration changes

### PR DESCRIPTION
**Pull request by itamarst**
Based on `boto-config-FLOC-2028`.

### CODE REVIEW

**Comment by exarkun** [comment]
This is broken, probably entirely.

**Comment by exarkun** [comment]
I think this works now.  There are no automated tests, of course...

File: flocker/node/agents/ebs.py [hunk 1]
```diff
@@ -21,32 +23,49 @@
...
+    """
+    Make boto log activity using Eliot.
+    """
+    logger = logging.getLogger("boto")
+    logger.addHandler(EliotLogHandler())
 
+    # It seems as though basically all boto log messages are at the same
+    # level.  Either we can see all of them or we can see none of them.
+    # We'll do some extra filtering in the handler.
+    logger.setLevel(logging.DEBUG)
```
Comment (myechuri) [inline flocker/node/agents/ebs.py]: When i set boto log level to ``logging.DEBUG``, the amount of information output was way too much and not so useful for debuggability. So, reverted to ``logging.ERROR``.

Comment (itamarst) [inline flocker/node/agents/ebs.py]: I believe this is handled by the logic in `EliotLogHandler`.

**Comment by itamarst** [comment]
1. Test logs show the Boto messages.
2. The failing test is a known failure.

**Comment by itamarst** [comment]
Thanks for doing this. Will merge now.

**Comment by itamarst** [comment]
Oh, actually, lint failures. Fix those, then merge.

**Comment by itamarst** [comment]
Superceded by some other branches, although we might still want to reuse some of it.

**Comment by itamarst** [comment]
Gah, wrong branch.

**Pull request merged**